app/helpers/sessions_helper.rb

```diff
@@ -57,4 +57,21 @@ def redirect_back_or default
   def store_location
     session[:forwarding_url] = request.original_url if request.get?
   end
+
+  def check_authenticate? user
+    user&.authenticate params[:session][:password]
```

Comment:
tách ra method riêng là gì e ?

Reply:
vì em check robocop bị lỗi quá nhiều tính toán trong một hàm nên em tách ra ạ

Reply:
có 1 dòng mà e, k cần tách đâu